hibernate-core/src/main/java/org/hibernate/sql/ast/tree/from/NamedTableReference.java

```diff
@@ -109,7 +105,7 @@ public TableReference getTableReference(
 
 	@Override
 	public String toString() {
-		return getTableExpression() + "(" + getIdentificationVariable() + ')';
+		return tableExpression + "(" + getIdentificationVariable() + ')';
```

Comment:
I think I'd prefer to see the `prunedTableExpression` if I click on `toString()` for such an object in the debugger. Please revert this.

Reply:
Done.